src/dbwrapper.h

```diff
@@ -17,6 +17,9 @@
 #include <leveldb/db.h>
 #include <leveldb/write_batch.h>
 
+static const size_t DBWRAPPER_PREALLOC_KEY_SIZE = 64;
+static const size_t DBWRAPPER_PREALLOC_VALUE_SIZE = 1024;
```

Comment:
I did benchmark the change overall,  and the "Flush to disk" time went down by around 5% from just this. I did not benchmark any other sizes, but my expectation is that as long as the key size is over 35, and the value over 100 or so, there will not be any measurable difference, as these cases occur so infrequently, and a vector resize isn't the end of the world.